### ISSUE TITLE
Exporting to EDEngineer Fails

### ISSUE THREAD

**Issue by rdc000** [EDCD/coriolis#582]
Reported error:
Uncaught Error: .field(name, val) val can not be empty
Browser: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/81.0.4044.122 Safari/537.36
Path: /outfit/alliance_chieftain?code=A4pktkFflndpsff4-2b1g292929040400-B15i-m3m32b2b2b25.Iw18RQ%3D%3D.IwBhrSumyA%3D%3D.H4sIAAAAAAAAA42Sr07DUBTGT7t1Y7Rr07I%2F3cKfwgokS3gADJkjGEKCADmFIAgECQgECBxiAUIwJBM8ABJBUDwAD4BAYiAoBIxzON8SrsG04svXnt895%2Fa7l7hIRD%2BOyuBMxe1ZROUjjyjcVxfduUTpvU0kFi%2BBzKmMvAxEgs4UUS3R9%2Bg2r3Wba6bTnkqQfolUJr9Fqv2AKAbU2q0rmeOmIQ%2F%2ByOGkymFVyWetiMOLBjpWKXksMnoREk3DzcDNws3BSYnXDX6t4nRLRIWdd5G48ybS6r2KLs8CubwByAa0pdKuqETYU7ypHxv4GfGyQOUskM%2FLZk9XOICbgiayprGFcPGjbjGFkyAzGfGqIbcRMCaFINPhAYzxCup51E9ULCwt9j9F5p%2F0OiTdD82hzm3T5BLjHtT6ybh2govQM4WTODPZyEw2ecGQ5yAn9Mr5yC6ES0%2BRndC%2F5xfs%2F4bE0AIAAA%3D%3D.IwelBYQBjA2aQFMCGBzANokYq4TKIA%3D%3D&bn=Chef%20Temp
Error: Unknown
Details:
scriptUrl: https://coriolis.io/main.js
line: 18
col: 356993
error: {}

This seems to happen only with certain build configurations. For example this one works fine: 
https://coriolis.io/outfit/imperial_courier?code=A0pat5F5l3das5f3270s2a040404041NB4m3273w24--.Iw18SQ%3D%3D.CwRgDOFtEnDMcTGNIA%3D%3D.H4sIAAAAAAAAA52RsUrDYBSFj00T26RNm9LWVKRSGzt0chJcxEXo4uTQxU1Q6NRBcRJ0EV%2FATj6CRUEEH8AXcCui%2BASCk4PW%2F3qu0B8dXAzkcLh89%2BTe3CmzCuAzTRkfUkprHlAYtoDqkwNk5sYikjJVhVyF9iiF5F0kvmJTuf4hUhkUgGZ%2FBhDHzFryQONugwn%2ByGK5UybkmpUJtJgzIq3rIpBV56trqFtQJxmza%2BP4wr2sAd5Fnp%2FTuPQWgyVrNhVyFKpT4uc3EbfBPbweXXP4Qsg3XZu0MxnseyYtoXbKjfE6Emlv00hgOhY%2Fp8wrFPU5WHLsA4E6yZn1X1BpaZ%2FLamZ8l2XPKGSPNiYqgYrk%2F9ET%2Fj3MiR2maDYs1KNU7pmUink%2BTw8VPfD%2FJDfTxJfppGTa9u4DneMoB4Rd3jA6YzFRCVRE8PP5AkMyhXUwAgAA&bn=T-16%20Back%20Home

**Comment by rdc000** [comment]
The issue seems to be tied to the "Plasma Slug" Exp Effect on the Plasma Accelerator module. Removing that causes the error to no longer appear.

**Comment by alterNERDtive** [comment]
Yup, can confirm. Exporting https://s.orbis.zone/a76O throws an error, removing the plasma slug works “fine” (still missing bulkhead and life support engineering, see #607).

**Comment by alterNERDtive** [comment]
```
Browser: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/86.0.4240.183 Safari/537.36
Path: /outfit/krait_phantom?code=A4pptkFflidussf5ApAp1g1g04040404B15j371O29m56gm325.Iw18WQ%3D%3D.CwBjIRgkGY5iAmZVijKIA%3D%3D%3D.H4sIAAAAAAAAA42SvUoDURCFJ78mm002d0nWNSSiZjUgKFoKFqlURETwBbQVixSCKVIkjVWKIMEqQgofwNLCUqwsfACxlBRqo4honHEmsBcEi91iOOz55gz3zgUcA4CfGJfhKZdUJwSQbpoA6piVfZUC8K7DABTCqiYbXBK9JyL7ntsd74uIwuho%2F4iLJX9zk99E%2Bb4F4F5GAcq1cU6KYEGTdZ8cTco18kw%2BsENRXPYhq%2F7M%2Ft0jD4nhim5tcUmaSGR0FcC0qBlRs6LmRFEc933czlYAFjfeiSa2SmwlcFcnnXOJ7SUB4oevRG71hajcGfC4ZBDICAKlgkAmrmuoJ8u4iPPpd%2FgKlSj3hns8UZQOTGYCkxau%2FSVrfI%2BqbbAvirK4rf0DWYusVUmSN1qrwk3xo%2BKfcAk134ispU8%2BogypDCL8APofnJTDeZ10JpNuWWamihwnypZgTxQ5WPQzTVmgK1ZhVUJcXNAhXQkpDYky8naUKK%2FNJBH88%2F0CHhi%2Fq%2FQCAAA%3D.EweloBhBmSQQwHbwMYHtEBN4hARgkJChKA%3D%3D
Error: Unknown
Details:
scriptUrl: https://coriolis.io/main.js
line: 18
col: 357155
error: {}
```

**Comment by CuanticoMC** [comment]
Fail to eport from coriolis to EDEngineer using Chrome and Edge

Browser: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/91.0.4472.77 Safari/537.36
Path: /outfit/alliance_chieftain?code=A4pktkFflndpsif42b2b1g29292904040002B12d3227m32b2b2b66.Iw18RQ%3D%3D.Aw1%2FgjBQ.H4sIAAAAAAAAA42Tv0sCcRjGXzVNvdPz1NNTJFLPJKN2l9ZaAregIaKhLWgIcgiqJRoaoqGpoKE%2FoKkawqk%2FoLEgmhsam8q%2Bb88r%2BUWl4G54eOD74Xl5fxypcSL6DkO6pxDjJECU2DeJ7B249J1B5N0HiTig5jW5B4k%2BfzFbh9NEufoYyEcEcVDlNLQNsbxP5uwESOfSInKvQVa38iBDqqjJdp%2FslcvuOiBf8MJh1dTQASRmKub4eoqoLK4iriZuShxH1JrgIYlb%2FEBc%2B515ToI5qpZ10gWksUq%2F3fUqFW7KRDMiHBslV4ZJswpShOMD5LGQrWEyVQcpwsYAufEPyaYfKOEHSqoFDZ3LUq8imEgLW7DFuQ8xLFUcW75JW9X6s6WjDGpKpZI4TqslHbIJCbpd5ois2miCt18hzlNSDkluJKMaGj%2BTmrewyckSSHFpORRPHDtqdpisIDgpfdrivDZIzo%2FGdXRcR8eJY9c3WfBNFlVJSPwqZPYu%2FA3NFmUAzPTn9wMX%2B7CWcgMAAA%3D%3D.EweloBhBGA2EoFMCGBzANokMK4fiIA%3D%3D&bn=PvE%20Cuantico%20Chieftain
Error: Unknown
Details:
scriptUrl: https://coriolis.io/main.js
line: 18
col: 348961
error: {}

**Comment by Sid127** [comment]
This is being addressed along with #705

**Comment by Sid127** [comment]
@felixlinker this can be closed
